From 03206dfd6415257627a055347fdd0c3cb0f5372f Mon Sep 17 00:00:00 2001
From: Carl Schwan <carl@carlschwan.eu>
Date: Mon, 23 Jun 2025 19:41:11 +0200
Subject: [PATCH] Do not request password twice

Otherwise the second time will crash as the password was already emitted
and the job deleted


(cherry picked from commit 25202045186262a081c960461a8b791f84fccb5c)

Co-authored-by: Carl Schwan <carl@carlschwan.eu>
---
 resources/imap/settingspasswordrequester.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/resources/imap/settingspasswordrequester.cpp b/resources/imap/settingspasswordrequester.cpp
index 28891e4e7b..f82d16a397 100644
--- a/resources/imap/settingspasswordrequester.cpp
+++ b/resources/imap/settingspasswordrequester.cpp
@@ -55,7 +55,6 @@ void SettingsPasswordRequester::requestPassword(RequestType request, const QStri
                 m_readPasswordJobs.removeAll(job);
             });
             m_readPasswordJobs << job;
-            job->start();
         } else {
             auto password = m_resource->settings()->password();
             if (password.isEmpty()) {
-- 
GitLab

